pkg/provision/scaleway.go

```diff
@@ -143,6 +155,49 @@ func (s *ScalewayProvisioner) Delete(id string) error {
 	return nil
 }
 
+// List returns a list of exit nodes
+func (s *ScalewayProvisioner) List(filter ListFilter) ([]*ProvisionedHost, error) {
+	var inlets []*ProvisionedHost
+	page := int32(1)
+	for {
+		servers, err := s.instanceAPI.ListServers(&instance.ListServersRequest{Page: &page})
+		if err != nil {
+			return inlets, err
+		}
+		for _, server := range servers.Servers {
+			for _, t := range server.Tags {
+				if t == filter.Filter {
+					host := &ProvisionedHost{
+						IP:     server.PublicIP.Address.String(),
+						ID:     server.ID,
+						Status: server.State.String(),
+					}
+					inlets = append(inlets, host)
+				}
+			}
+		}
+		if len(servers.Servers) < 20 {
```

Comment:
What is this doing?

Reply:
Where does the magic number 20 come from and what does it mean?

Reply:
> What is this doing?
If the returned items is less than the default page size then I exit.

> Where does the magic number 20 come from and what does it mean?
It is the default page size for the api, I will configure it as a variable to make it clearer.